drivers/hmis/spec/jobs/auto_exit_job_spec.rb

```diff
@@ -54,6 +54,20 @@
       Hmis::AutoExitJob.perform_now
       expect(e1.exit).to have_attributes(auto_exited: be_present, exit_date: e1.entry_date, destination: 30)
     end
+
+    it 'should not fail if enrollment has contact date before entry (regression #7178)' do
+      e1 = create :hmis_hud_enrollment, data_source: ds1, project: p1, client: c1, user: u1, entry_date: Date.current - 2.months
```

Comment:
just as a random note, it's not necessary to pass `user: u1`. It will be generated using the factory if not passed explicitly. We do have a lot of tests floating around that specify the user, but its not necessary unless we're trying to test something specific about that user.